interface/src/admin/agenda/client.ts

```diff
@@ -1,9 +1,6 @@
+/* eslint-disable @typescript-eslint/no-redeclare -- A Zod schema name and type should have the same names */
```

Comment:
[@typescript-eslint/no-redeclare](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-redeclare.md) 규칙이 의도적으로 Zod 스키마와 타입을 같은 이름으로 선언하는 것을 금지한다고 합니다.
> **Note**: Even with this option set to true, this rule will report if you name a type and a variable the same name. **_This is intentional._** Declaring a variable and a type and a variable the same is usually an accident, and it can lead to hard-to-understand code. If you have a rare case where you're intentionally naming a type the same name as a variable, use a disable comment.

위와 마찬가지로 이 규칙을 프로젝트 전역에서 비활성화하기에는 이 규칙의 이점이 너무 많아서, interface쪽 파일들에서만 규칙을 껐습니다.